<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PortSettingsDialog</class>
 <widget class="QDialog" name="PortSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="selectBox">
     <property name="title">
      <string>Serial port configuration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="6" column="0">
       <widget class="QLabel" name="flowControlLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Flow control</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QComboBox" name="parityBox"/>
      </item>
      <item row="6" column="2">
       <widget class="QComboBox" name="flowControlBox"/>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="serialPortBox"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="stopBitsLabel">
        <property name="text">
         <string>Stop bits</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QComboBox" name="baudRateBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="serialPortLabel">
        <property name="text">
         <string>Serial port</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QComboBox" name="dataBitsBox"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="parityLabel">
        <property name="text">
         <string>Parity</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="baudRateLabel">
        <property name="text">
         <string>Baud rate</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="dataBitsLabel">
        <property name="text">
         <string>Data bits</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QComboBox" name="stopBitsBox"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="portsInfoButton">
        <property name="text">
         <string>Show</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Ports info</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PortSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PortSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
